### ISSUE TITLE
Simplex Promo

### ISSUE THREAD

**Issue by dternyak** [MyCryptoHQ/MyCrypto#1785]
- [ ] Add rotating promo for Simplex.
     - link to buy.mycrypto.com
     - Determine geolocation on client, and only show on non-US countries
     - "Buy Ether with Euro" @jspence425 to confirm messaging

**Comment by james-prado** [comment]
I can jump on this and revamp the promo carousel. @jspence425 were you just of moving it on the page so it's more visible to the user, or something else?

**Comment by james-prado** [comment]
I've run into some blocking issues with this. 

1. There really isn't any good way to determine if a user is outside the us without using either the [geolocation api](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation) which requires asking the user for permission, or using a 3rd party api to do IP address lookup, which seems very excessive just for a simplex promo.
2. Visually, I'm really not sure what can be done to make the promos a bit more interesting. I'm pretty limited by the colours and the assets available by simplex. Right now it just looks like this.
<img width="644" alt="screen shot 2018-05-22 at 6 00 52 am" src="https://user-images.githubusercontent.com/13836920/40355909-a4cfd778-5d85-11e8-81ed-0e0a34998947.png">

I've tried moving the promos around, and seeing if I could get it to work somewhere else without cramming it between summary & tokens, but tbh I found it to be the least bad option.

**Comment by james-prado** [comment]
@dternyak I've run into some blocking issues with this. 

There really isn't any good way to determine if a user is outside the us without using either the [geolocation api](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation) which requires asking the user for permission, or using a 3rd party api to do IP address lookup, which seems very excessive just for a simplex promo.

**Comment by 409H** [comment]
You could check the Accept-Language header for en-us to determine it and assume everyone has their language settings correct - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language

**Comment by james-prado** [comment]
I tried using the browser local - `navigator.language`, but that isn't really specific enough, there are lots of exceptions. If we show to all french and spanish users, we might be trying to tell Quebecers and Mexicans to buy crypto with ETH, which makes no sense. Then I tried formatting a number to the local currency and checking if it uses euros, but that doesn't work because you have to manually specify the currency for whatever locale your supporting.

the below doesn't work because you have to manually specify `{style: 'currency', currency: 'EUR' }`

`(12).toLocaleString(navigator.language, { style: 'currency' })`

timezones might be the closest, but it's still kinda inaccurate. You could specify a range of timezones, but Italy, Sweden, and parts of Africa are all on GMT+2, but Sweden and Libya don't use euros.

**Comment by dternyak** [comment]
@james-prado Agreed, we don't want to ask users for permission for location just for this feature, it's pretty intrusive.

I'd like to test out the timezone approach here as well. Absolute accuracy is not a requirement, we mostly just don't want to show the option for US users as they will not be able to actually use simplex.

**Comment by tayvano** [comment]
I agree. 

We can also update language on the promos to be BUY ETH WITH USD and BUY ETH WITH EUR and leave both promos on if timezones don't work out.
